### PR TITLE
[ADD] account_asset_depr_line_show_move

### PR DESCRIPTION
**Pull request by pedrobaeza**
# 
# Button for showing the move on account asset depreciation line

This module adds a button to show the journal entry associated with an asset
depreciation line.
# Usage

On each depreciation line that has created a journal entry, you have a new
button with an arrow that allows you to navigate to the journal entry.

### CODE REVIEW

File: account_asset_depr_line_show_move/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+    'name': 'Assets Management - Show move button',
+    'version': '8.0.1.0.0',
+    'depends': ['account_asset'],
+    'author': "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.serviciosbaeza.com',
+    'category': 'Accounting & Finance',
+    'test': [
+    ],
```
Comment (sergio-incaser) [inline account_asset_depr_line_show_move/__openerp__.py]: Remove empty keys.

File: account_asset_depr_line_show_move/views/account_asset_asset_view.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="view_account_asset_asset_form">
+            <field name="name">Account Asset Form (cancel button)</field>
```
Comment (sergio-incaser) [inline account_asset_depr_line_show_move/views/account_asset_asset_view.xml]: First id and on second place the model

File: account_asset_depr_line_show_move/views/account_asset_asset_view.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="view_account_asset_asset_form">
```
Comment (sergio-teruel) [inline account_asset_depr_line_show_move/views/account_asset_asset_view.xml]: First id and on second place the model

File: account_asset_depr_line_show_move/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+{
+    'name': 'Assets Management - Show move button',
+    'version': '8.0.1.0.0',
+    'depends': ['account_asset'],
+    'author': "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.serviciosbaeza.com',
+    'category': 'Accounting & Finance',
+    'test': [
```
Comment (sergio-teruel) [inline account_asset_depr_line_show_move/__openerp__.py]: Remove empty keys

**Comment by pedrobaeza** [comment]
@sergio-incaser, thanks for the review. Changes done.

**Comment by pedrobaeza** [comment]
The coverage decrease is due to https://github.com/OCA/maintainer-quality-tools/issues/249

**Comment by rvalyi** [comment]
:+1:  LGTM.
Question: why did you have to change the .travis.yml? Is that because of the coverage issue?

**Comment by pedrobaeza** [comment]
It's due to the module _account_asset_management_ module, which is a fork of the core _account_asset_ module, and makes incompatible the use of both. I was trying the first one, and I decide to extend the core one with the functionalities I want from the *_management one, but don't use it because of the problems a fork means (and more now that v9 comes with new features for assets), and technically the module is bad (basically because a lot of SQL).

Runbot still fails because it tries to install both, but it doesn't include an option for excluding modules. The sooner we forget this module fork, the better...

**Comment by luc-demeyer** [comment]
@pedrobaeza 
if you want to fix the standard 'account_asset' module than you'll end up after a few weeks of work with the same functionality as what we have implemented in the 'account_asset_management' module. I think it is better that we focus our energy on common initiatives. 
Concerning the remark that the 'account_asset_management' module is bad because of a lot of SQL: 
please have a look at the standard 'account_asset' module and you'll also see a lot of SQL.

**Comment by pedrobaeza** [comment]
The main reason is not having an incompatible module installed that compromise the migration path. I know core code is also not good enough, but it's something that I don't have to maintain, and in v9 all things change, so I prefer to go with _account_asset_ + extensions (the propose 2 + the disposal, that is coming), and then go to the core v9 _account_asset_.

**Comment by rvalyi** [comment]
I personally don't see any issue adding this small and clean module in the 8.0 branch even if we have an incompatibility that we already had as far as I understand.

**Comment by pedrobaeza** [comment]
Can anyone review this PR?

**Comment by antespi** [comment]
:+1: Works fine in my local instance

**Comment by pedrobaeza** [comment]
cc @Tecnativa

**Comment by rafaelbn** [comment]
@pedrobaeza please review runbot and coverage, thanks!

**Comment by pedrobaeza** [comment]
Moved to 10.0.

**Pull request closed without merging**